extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsHooks.class.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class EmbeddableDiscussionsHooks {
+
+	public static function onParserFirstCallInit( Parser $parser ) {
+		global $wgEnableDiscussions;
+
+		if ( $wgEnableDiscussions ) {
+			$parser->setHook( EmbeddableDiscussionsController::TAG_NAME, [
+				'EmbeddableDiscussionsController',
+				'render'
+			] );
```

Comment:
Now that render is non-static, I think this should be:

$parser->setHook( EmbeddableDiscussionsController::TAG_NAME, [ new EmbeddableDiscussionsController(), 'render' ] );